test/tools/ossfuzz/strictasm_diff_ossfuzz.cpp

```diff
@@ -44,6 +44,9 @@ using namespace solidity::util;
 using namespace solidity::langutil;
 using namespace solidity::yul::test::yul_fuzzer;
 
+// Prototype -- there's no header available.
+extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size);
```

Comment:
Hm... that's a bit silly...

Reply:
Depends on the answer to https://github.com/ethereum/solidity/pull/10537/files#r538879939 I just saw, I guess :-).

Reply:
It seems @bshastry did some interesting hack in `SolidityCustomMutatorInterface.cpp`: moved these `extern "C"` function into an anonymous namespace.